components/scheduler/README.md

```diff
@@ -0,0 +1,143 @@
+# Nylas scheduler
+
+Nylas scheduler (`<nylas-contact-list>`) is part of the Nylas Components library that lets you build user-facing contact functionality into your application in minutes. Use Nylas scheduler with your Nylas account or by passing in your own JSON data.
```

Comment:
`Nylas scheduler (`<nylas-contact-list>`)`? Should this be `(`nylas-scheduler`)`?